analysis/analyze-hotspots/README.md

```diff
@@ -1,31 +1,33 @@
-# Analyze Hotspots
+# Analyze hotspots
 
-Perform hotspot analysis using a geoprocessing service.
+Use a geoprocessing service and a set of features to identify statistically significant hot spots and cold spots.
 
-In this case, frequency of 911 calls in an area are analyzed.
+![Image of analyze hotspots](AnalyzeHotspots.png)
 
-![](AnalyzeHotspots.png)
+## Use case
+
+This tool identifies statistically significant spatial clusters of high values (hot spots) and low values (cold spots). For example, a hotspot analysis based on the frequency of 911 calls within a set region.
 
 ## How to use the sample
 
-Select a start and end date using the datepickers between 1/1/1998 and 5/31/1998 respectively. Click the "Analyze hotspots" button to start the geoprocessing job.
+Select a date range (between 1998-01-01 and 1998-05-31) from the dialog and click Analyze. The results will be shown on the map upon successful completion of the `GeoprocessingJob`.
```

Comment:
As a team we decided on keeping class names out of how to use the sample section and keeping it plain english (going by the wiki), though before that change was agreed a few readmes had already been approved. We caught a few of these in the Android readmes too. We replaced them with plain english in Android, I've changed it below along with reverting the button name to how we implement it in Java! 
```suggestion
Select a date range (between 1998-01-01 and 1998-05-31) from the dialog and click "Analyze Hotspots". The results will be shown on the map upon successful completion of the geoprocessing job.
```